clients/spark/core/src/main/scala/io/treeverse/clients/EntryRecord.scala

```diff
@@ -1,9 +0,0 @@
-package io.treeverse.clients
```

Comment:
We now use the original from lakefs.catalog, right?

Reply:
The project generated `Entry` class based on `catalog.proto` found in the project.
Didn't found any use of EntryRecord in the code, so I've removed it.